src/modules/HomePage/index.tsx

```diff
@@ -3,30 +3,77 @@ import { Link } from "react-router-dom";
 import Menu from "../Common/Menu";
 import MenuItem from "../Common/MenuItem";
 import userRoutes from "../../routes/user";
+import ActiveProfileTab from "../../assets/Tabs/Tab_UserProfile_Active.png";
+import ActiveQuestsTab from "../../assets/Tabs/Tab_NewQuests_Active.png";
+import ActiveQuestLogTab from "../../assets/Tabs/Tab_QuestLog_Active.png";
 import ProfileTab from "../../assets/Tabs/Tab_UserProfile_Inactive.png";
 import QuestsTab from "../../assets/Tabs/Tab_NewQuests_Inactice.png";
 import QuestLogTab from "../../assets/Tabs/Tab_QuestLog_Inactive.png";
+import { HomePageProps } from "../../interfaces";
 import "./HomePage.scss";
 
-interface HomePageProps {
-  children?: object;
-}
-
-const HomePage: React.FC<HomePageProps> = (props) => {
-  const { children } = props;
-
+const HomePage: React.FC<HomePageProps> = ({
+  activePage,
+  setActivePage,
+  children,
+}) => {
   return (
     <section className="homepage">
       <section className="tabs-grid">
-        <Link className="profile-tab" to={userRoutes.profile.path}>
-          <img className="tab" src={ProfileTab} alt="" />
-        </Link>
-        <Link className="quests-tab" to={userRoutes.availableQuests.path}>
-          <img className="tab" src={QuestsTab} alt="" />
-        </Link>
-        <Link className="quest-log-tab" to={userRoutes.userQuestLog.path}>
-          <img className="tab" src={QuestLogTab} alt="" />
-        </Link>
+        {activePage === "profile" && (
```

Comment:
Nice work with tabs! I think if we have time this part can be refactored to avoid duplicated elements, and maybe use ternary to simplify it

Reply:
That's a fantastic Idea, I'll try that